src/time_steppers.jl

```diff
@@ -198,30 +198,30 @@ function adams_bashforth_update_source_terms!(grid::Grid, Gu, Gv, Gw, GT, GS, Gp
 end
 
 "Store previous value of the source term and calculate current source term."
-function calculate_source_term_divergence!(::CPU, grid::Grid, Gu, Gv, Gw, RHS)
+function calculate_source_term_divergence!(::CPU, grid::Grid, Δt, u, v, w, Gu, Gv, Gw, RHS)
```

Comment:
The name of this function should be changed.

Reply:
True. I went with `calculate_poisson_right_hand_side`.